lib/Runtime/Library/JavascriptArray.inl

```diff
@@ -129,6 +131,12 @@ namespace Js
         {
             segmentMap->SwapSegment(seg->left, seg, newSeg);
         }
+
+        if (isInlineSegment)
+        {
+            this->ClearElements(seg, 0);
+        }
```

Comment:
correct.

---
In reply to: [123792263](https://github.com/Microsoft/ChakraCore/pull/3211#discussion_r123792263) [](ancestors = 123792263)